docs/contribute.md

```diff
@@ -1,111 +1,35 @@
+1. **Prerequisites**
+   - [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+   - [Ruby 3](https://www.ruby-lang.org/en/documentation/installation/)
 
-3. **Clone the Forked Repository**
+1. **Clone the Forked Repository**
 
    ```bash
    git clone https://github.com/YOUR_USERNAME/LinksForIsrael.git
    cd LinksForIsrael
    ```
 
-4. **(Optional: If you want to run the GitHub Pages project locally) Install Dependencies**
-   - Install bundler:
-
-     ```bash
-     gem install bundler
-     ```
-
-   - Install project dependencies:
-
-     ```bash
-     bundle install
-     ```
-
-5. **Run Locally**
-
-   ```bash
-   bundle exec jekyll serve
-   ```
-
-   > ⚠️ **Note**: There is currently an issue with running the project locally. The theme doesn't display as expected. If you can solve this issue, your contribution would be greatly appreciated!
+1. **Install Dependencies**
+   - Install bundler: `gem install bundler`
+   - Install project dependencies: `bundle install`
 
-6. **Install Ruby 3**
-   Check how to install an up-to-date version of Ruby for your system [here](https://www.ruby-lang.org/en/documentation/installation/).
+1. ** Setup GitHub API token **
+   - Generate token: https://github.com/settings/tokens (check very `read` permission)
+   - Save the token in some secure location (e.g. [BitWarden](https://bitwarden.com/))
 
-7. **Install Ruby dependencies**
-   Bundler should be installed now as well. Run `bundle install`.
+1. **Run Locally**
 
-8. **Run the project locally**
    ```bash
+   export JEKYLL_GITHUB_TOKEN=<your GitHub API token from the step above>
    bundle exec jekyll serve
    ```
 
-9. **Run the project locally on a mobile device**
-   Make sure your phone is connected to the same network as the server
-   Run serve with the `--host` flag:
+1. **Run the project locally on a mobile device**
+   Make sure your phone is connected to the same network as the server and add the `--host` flag to the command above:
    ```
    bundle exec jekyll serve --host=0.0.0.0
    ```
-   On your mobile device open your server's local IP (you can get it using tools like `ipconfig getifaddr en0`)
-
-## 📖 Understanding the Project Structure
-
-Take some time to understand the layout of the repository. Familiarize yourself with the directory structure, the type of files, and their content.
-
-Then, pick one of the open issues in [*Github Issues*](<https://github.com/users/4tal/projects/2/views/2>)
-
-## 🛠 Making Changes
-
-1. **Create a New Branch**
-   ```bash
-   git checkout -b my-new-feature
-   ```
-
-2. **Find the Appropriate Page/Section**
-   - Navigate to the file where you want to add or modify content.
-
-3. **Make Your Changes**
-   - Edit the files using your preferred code editor.
-   - Remember to follow the project's coding standards and maintain the existing style.
-
-4. **Commit Your Changes**
```

Comment:
I know it's funny to have this in the guideline (as it's not a git tutorial), but I think because this is open-source and the levels of experience by the different potential contributors might differ I think it's better to have it here (I was considering adding a short explanation about how to work with upstream remotes, which I have to Google every time I start working on a new fork)

Reply:
you are only proving my point... these are all standard things anyone can find with a simple google search
Example I found in 3 seconds: https://github.com/firstcontributions/first-contributions